src/Resources/app/administration/src/components/cko-modal/index.js

```diff
@@ -1,168 +1,168 @@
-import template from './cko-modal-html.twig';
-import './cko-modal.scss';
+import template from "./cko-modal-html.twig";
+import "./cko-modal.scss";
 
 const { Component, Service } = Shopware;
-const HTTP_STATUS_CODE_200 = 202
+const HTTP_STATUS_CODE_202 = 202;
+const HTTP_STATUS_CODE_200 = 200;
 
-Shopware.Component.register('cko-modal', {
-    template,
-    inject: [
-        'CkoVoidService',
-        'CkoCaptureService',
-        'CkoRefundService'
-    ],
-    data() {
-        return {
-            //
-        };
+Shopware.Component.register("cko-modal", {
+  template,
+  inject: ["CkoVoidService", "CkoCaptureService", "CkoRefundService"],
+  data() {
+    return {
+      //
+    };
+  },
+  props: {
+    isLoading: {
+      type: Boolean,
+      required: true,
     },
-    props:{
-        isLoading: {
-            type: Boolean,
-            required: true
-        },
-        action:{
-            type: String,
-            required: true,
-            default:''
-        },
-        orderInfo: {
-            type: Object,
-            required: true
-        },
-        paymentMethod: {
-            type: String,
-            required: true,
-            default: ''
+    action: {
+      type: String,
+      required: true,
+      default: "",
+    },
+    orderInfo: {
+      type: Object,
+      required: true,
+    },
+    paymentMethod: {
+      type: String,
+      required: true,
+      default: "",
+    },
+  },
+  methods: {
+    closeModal() {
+      this.$emit("modal-close");
+    },
+    captureOrder(amountCapture, paymentMethod) {
+      this.CkoCaptureService.capture({
+        payment_id: this.orderInfo.customFields.payment_approved.payment_id,
+        amount: amountCapture,
+        currency: this.orderInfo.currency.isoCode,
+        payment_method: paymentMethod,
+      }).then((response) => {
+        console.log(response);
+        if (
```

Comment:
Remove `console.log`, and refactor status checks (and page reload) to a common hander.